test/index.js

```diff
@@ -278,6 +278,48 @@ describe('koa-body', async () => {
     });
   });
 
+  describe('onError option', () => {
+    it('onError should not capture errors thrown in downstream middleware', async () => {
+      let koaBodyError;
+      app.use(async (ctx, next) => {
+        try {
+          await next();
+        } catch (err) {
+          ctx.status = 201;
+        }
+      });
+      app.use(koaBody({
+        onError: (err, ctx) => {
+          koaBodyError = err;
+        }
+      }));
+      app.use(async (ctx, next) => {
+        ctx.status = 200;
+        throw new Error();
+      });
+
+      await request(http.createServer(app.callback()))
+        .get('/test')
+        .expect(201);
+
+      expect(koaBodyError).to.be.undefined;
```

Comment:
this needs to be a function call:
```
expect(koaBodyError).to.be.an('undefined');
```

Reply:
this is actually an issue on the `v5` branch

Reply:
Linter didn't yell at me, not my fault!  :p